src/cookies.njk

```diff
@@ -28,44 +29,113 @@ layout: layout-single-page.njk
-              legend: {
-                text: "Do you want to accept analytics cookies?"
-              },
-              classes: "js-cookies-page-form-fieldset",
-              attributes: {
-                id: "analytics",
-                hidden: true
-              }
-            },
-            items: [
-              {
-                text: "Yes",
-                value: "yes"
-              },
-              {
-                text: "No",
-                value: "no"
-              }
+        <h2 class="govuk-heading-l govuk-!-margin-top-6">Essential cookies (strictly necessary)</h2>
+
+        <p class="govuk-body">We use an essential cookie to remember when you accept or reject cookies on our website.</p>
+
+        {{ govukTable({
+          firstCellIsHeader: true,
+          caption: "Essential cookies we use",
+          head: [
+            { text: "Name" },
+            { text: "Purpose"},
+            { text: "Expires"}
+          ],
+          rows: [
+            [
+              {text: "design-system-cookie-policy"},
+              {text: "Saves your cookie consent settings"},
+              {text: "1 year"}
+            ]
+          ]
+        }) }}
+
+        <h2 class="govuk-heading-l govuk-!-margin-top-6">Analytics cookies (optional)</h2>
+
+        <p class="govuk-body">We use Google Analytics software to understand how people use the GOV.UK Design System. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.</p>
+
+        <p class="govuk-body">We do not collect or store your personal information (for example your name or address) so this information cannot be used to identify who you are.</p>
+
+        <p class="govuk-body">We do not allow Google to use or share our analytics data.</p>
+
+        <p class="govuk-body">Google Analytics stores information about:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>the pages you visit</li>
+          <li>how long you spend on each page</li>
+          <li>how you arrived at the site</li>
+          <li>what you click on while you visit the site</li>
+          <li>the device and browser you use</li>
+        </ul>
+
+        {{ govukTable({
+          firstCellIsHeader: true,
+          caption: "Analytics cookies we use",
+          head: [
+            { text: "Name" },
+            { text: "Purpose" },
+            { text: "Expires" }
+          ],
+          rows: [
+            [
+              { text: "_ga" },
+              { text: "Used to distinguish users." },
+              { text: "2 years" }
+            ],
+            [
+              { text: "_gid" },
+              { text: "Used to distinguish users." },
+              { text: "24 hours" }
+            ],
+            [
+              { text: "_gat" },
+              { text: "Used to throttle request rate. If Google Analytics is deployed via Google Tag Manager, this cookie will be named _dc_gtm_<property- id>." },
```

Comment:
The user's not going to know whether this is deployed via GTM, or what the property ID is – but we do, so we should be able to provide an accurate cookie name?

```suggestion
              { text: "_dc_gtm_GTM-53XG2JT" },
              { text: "Used to throttle request rate." },
```

(I've not verified the exact cookie name)

Reply:
@36degrees I may be wrong, but I think the 'property ID' that the Google docs refer to might actually refer to the Google Analytics property ID rather than the GTM one. So the cookie name would look like this: "_dc_gtm_UA-12345678". Because we're switching out the Google Analytics property ID dynamically, I'm not sure if we'd want to hardcode this in case it changed behind the scenes.

Reply:
I think something like '_dc_gtm_UA-[random number]' would do for the cookie name, have made that change.